### PR TITLE
[OSK] Some enhancements for locales IT and RO

### PR DESCRIPTION
**Pull request by Fraizeraust**
## Purpose & Changes
Special thanks to Petr Akhlamov, this patch brings some improvements regarding the sizes of window elements of OSK (Virtual Keyboard). Note that this is still in progress as I'm continuing improving the Romanian locale.
# Images
Italian: https://imgur.com/a/iPjCF
Romanian: https://imgur.com/a/wnlYv
## TODO
- [X] Element improvements for Romanian locale
- [x] Test both locales for visual results (to ensure that everything is alright)

### CODE REVIEW

**Comment by Fraizeraust** [comment]
The PR is no longer WIP, it is now ready for merge.

**Comment by binarymaster** [comment]
@Fraizeraust you may want to rebase your changes on top of recent commits to make Appveyor happy.

File: base/applications/osk/lang/it-IT.rc [hunk 1]
```diff
@@ -1,129 +1,129 @@
 /*
- * PROJECT:         ReactOS Kernel
- * LICENSE:         GPL - See COPYING in the top level directory
- * FILE:            base/applications/osk/lang/it-IT.rc
- * PURPOSE:         On screen keyboard (Italian resources)
- * PROGRAMMERS:     Denis ROBERT
+ * PROJECT:     ReactOS Virtual Keyboard - OSK
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     Virtual Keyboard On screen (Italian resources)
+ * COPYRIGHT:   Copyright Denis ROBERT
```
Comment (learn-more) [inline base/applications/osk/lang/it-IT.rc]: You changed the license, so you need to ask all authors if this is ok.

File: base/applications/osk/lang/ro-RO.rc [hunk 2]
```diff
@@ -1,129 +1,129 @@
...
- * LICENSE:         GPL - See COPYING in the top level directory
- * FILE:            base/applications/osk/lang/ro-RO.rc
- * PURPOSE:         On screen keyboard (Romanian resources).
- * PROGRAMMERS:     Denis ROBERT
- * TRANSLATORS:     Ștefan Fulea (stefan dot fulea at mail dot md)
+ * PROJECT:     ReactOS Virtual Keyboard - OSK
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     On screen keyboard (Romanian resources)
+ * COPYRIGHT:   Copyright Denis ROBERT
+ *              Copyright Ștefan Fulea (stefan dot fulea at mail dot md)
```
Comment (learn-more) [inline base/applications/osk/lang/ro-RO.rc]: Ditto, you changed license so you need to have an OK from all authors

**Comment by Fraizeraust** [comment]
@binarymaster How do I do that?
@learn-more Ugh, are Denis and Stefan even active? That would be annoying for me waiting all the time if they do not respond (and if that is the case I'll just change the licence headers back and only add my name).

**Comment by learn-more** [comment]
@Fraizeraust If you just wanted to change the header, change the header without changing the actual license.

**Comment by Fraizeraust** [comment]
Here you go. Is it good now?

**Comment by Fraizeraust** [comment]
Anybody cares to merge my PR, please?

**Comment by binarymaster** [comment]
@Fraizeraust PR #412 is highly related to this one, and it seems it's in WIP state.

File: base/applications/osk/lang/it-IT.rc (hunk 1 above)
Comment (HBelusca) [reply base/applications/osk/lang/it-IT.rc]: And what do you do when the author is deceased/left without any traces/etc...?
Also, the COPYING in top-level directory looks like GPL-2 (with our addition too).

**Pull request merged**